bigtop.bom

```diff
@@ -145,11 +145,11 @@ bigtop {
     'hadoop' {
       name    = 'hadoop'
       relNotes = 'Apache Hadoop'
-      version { base = '3.3.3'; pkg = base; release = 1 }
+      version { base = '3.3.4'; pkg = base; release = 1 }
       tarball { destination = "${name}-${version.base}.tar.gz"
                 source      = "${name}-${version.base}-src.tar.gz" }
-      url     { download_path = "/$name/common/$name-${version.base}"
-                site = "${apache.APACHE_MIRROR}/${download_path}"
+      url     { download_path = "/$name/$name-${version.base}-RC1"
```

Comment:
Thanks for updating it. @iwasakims 
May I ask when would Hadoop upstream guys make the stable release for 3.3.4, or they prefer to offer release candidate 2 (RC2)?

Reply:
@guyuqi No one is willing to cut another RC before new issue is found on ongoing vote. There is no veto now.